### PR TITLE
Improved wording for novices regarding quality scores

### PR DESCRIPTION
**Pull request by hoytpr**
EDIT: peterjc has pointed out several problems with this pull request. Considering the age of the request, and the odd issues, I recommend this pull request be closed and I'll open another after re-cloning the repo. 

Have taught this before. Can be much more confusing that expected. This uses the previous example, adds the other, and changes some wording.

### CODE REVIEW

File: lessons/00-readQC.md [hunk 1]
```diff
@@ -46,21 +49,19 @@ As mentioned above, line 4 is a encoding of the quality. In this case, the code
...
  S - Sanger        Phred+33,  raw reads typically (0, 40)
  X - Solexa        Solexa+64, raw reads typically (-5, 40)
  I - Illumina 1.3+ Phred+64,  raw reads typically (0, 40)
  J - Illumina 1.5+ Phred+64,  raw reads typically (3, 40)
-     with 0=unused, 1=unused, 2=Read Segment Quality Control Indicator (bold) 
-     (Note: See discussion above).
  L - Illumina 1.8+ Phred+33,  raw reads typically (0, 41)
  ```
- So using the Illumina 1.8 encoding, which is what you will mostly see from now on, our first C is called with a Phred score of 0 and our Ns are called with a score of 2. Read quality is assessed using the Phred Quality Score.  This score is logarithmically based and the score values can be interpreted as follows:
+ So using the Illumina 1.8 encouding, which is what you will mostly see from now on, our first c is called with a Phred score of 0 and our Ns are called with a score of 2. Read quality is assessed using the Phred Quality Score.  This score is logarithmically based and the score values can be interpreted as follows:
```
Comment (peterjc) [inline lessons/00-readQC.md]: Typo

File: lessons/00-readQC.md [hunk 2]
```diff
@@ -187,11 +221,11 @@ Once we have an idea of the quality of our raw data, it is time to trim away ada
 
 Because *Trimmomatic* is java based, it is run using the command:
 
-**_java -jar trimmomatic-0.32.jar_**
+**_java jar trimmomatic-0.32.jar_**
 
 What follows this are the specific commands that tells the program exactly how you want it to operate. *Trimmomatic* has a variety of options and parameters:
 
-* **_-threads_** How many processors do you want *Trimmomatic* to run with?
+* **_-threds_** How many processors do you want *Trimmomatic* to run with?
```
Comment (peterjc) [inline lessons/00-readQC.md]: Typo?

File: lessons/00-readQC.md [hunk 3]
```diff
@@ -187,11 +221,11 @@ Once we have an idea of the quality of our raw data, it is time to trim away ada
 
 Because *Trimmomatic* is java based, it is run using the command:
 
-**_java -jar trimmomatic-0.32.jar_**
+**_java jar trimmomatic-0.32.jar_**
```
Comment (peterjc) [inline lessons/00-readQC.md]: You've changed ``java -jar ...`` to ``java jar ...`` here and elsewhere, which is wrong. A search and replace gone wrong?

File: lessons/00-readQC.md [hunk 4]
```diff
@@ -205,13 +239,13 @@ What follows this are the specific commands that tells the program exactly how y
 
 A generic command for *Trimmomatic* looks like this:
 
-**java -jar trimmomatic-0.32.jar SE**
+**java jar trimmomatic-0.32.jar SE -thr**
```
Comment (peterjc) [inline lessons/00-readQC.md]: In addition to the ``-jar`` issues noted above, why does this include what looks like a partial threads argument?

**Comment by peterjc** [comment]
There seems to be lots of typos and accidental changes in this pull request :(

File: lessons/00-readQC.md (hunk 4 above)
Comment (hoytpr) [reply lessons/00-readQC.md]: As per your comment below, there were errors converting over to Jekyll, and also during markdown conversions. The markdown conversions likely explain the loss of hyphens although I'm not sure exactly why yet. Also checking on the partial threads issue. Thanks for the feedback and I'll work on fixes immediately.

**Comment by peterjc** [comment]
As the creator, you should be able to close the pull request - or wait for one of the lesson owners.

See also #45 where I suggest removing the FASTQ encoding table.

**Pull request closed without merging**